lib/tasks/scheduler.rake

```diff
@@ -16,24 +16,12 @@ task :house_valuation_collector => :environment do
 end
 
 desc 'Ping Zillow to get property zpid'
-task :collect_zpids => :environment do
-  missing_zpids = ::HouseMetadatum.where(zpid: nil)
-  puts "Looking over #{missing_zpids.count} missing Zillow ids"
-  missing_zpids.each do |metadata|
-    house = metadata.house
-    address = house.address
```

Comment:
This was improved in the query above because of the following reasons:

1. We were loading each `HouseMetadatum` into memory -- presumably this isn't too bad, but if we ever have to ingest several neighborhoods at once with many houses, this can be a problem.
1. We were running into N+1 queries. Each `house_metadatum.house` call was invoking a `SELECT * FROM houses WHERE house_id = ?` query. If we were missing 300 zpids, this would result in 300 queries. Now we use `.includes` to perform a single left join (per batch).